svd/devices/atsamd51n20a.xsl

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It looked like this file was the same for each chip, so I just copied it for the new one. Is that correct?

Reply:
Yes, we just recently added the patching system so there are currently very few items. This looks correct.